openmrs/apps/appointments/printListView.html

```diff
@@ -1,32 +1,46 @@
-<title>Bahmni - Appointment Scheduling</title>
-<style>
```

Comment:
Why is this change done? Is this discussion documented ?

Reply:
It is a bug @buvaneswar-arun. There is nothing newly added here except copying from bahmniapps. Find more details in https://bahmni.atlassian.net/browse/BAH-1036

Reply:
@buvaneswari-arun you have anything pending on this PR to be approved ?